scripts/create_sql_lite_dump.py

```diff
@@ -0,0 +1,125 @@
+    "producedBy_responsibility": "produced_by_responsibility",
+    "producedBy_resultTime": "produced_by_result_time",
+    "producedBy_samplingSite_description": "produced_by_sampling_site_description",
+    "producedBy_samplingSite_label": "produced_by_sampling_site_label",
+    "producedBy_samplingSite_location_elevationInMeters": "produced_by_sampling_site_elevation_in_meters",
+    "producedBy_samplingSite_location_ll": "produced_by_sampling_site_location_ll",
+    "producedBy_samplingSite_placeName": "produced_by_sampling_site_place_name",
+    "registrant": "registrant",
+    "samplingPurpose": "sampling_purpose",
+    "curation_label": "curation_label",
+    "curation_description": "curation_description",
+    "curation_accessConstraints": "curation_access_constraints",
+    "curation_location": "curation_location",
+    "curation_responsibility": "curation_responsibility",
+    "relatedResource_isb_core_id": "related_resources_isb_core_id",
+    "source": "source"
+}
+
+NUMERIC_COLUMNS = ["produced_by_sampling_site_elevation_in_meters"]
+
+
+# Filter out any placeholder values that shouldn't appear in the dump
+def _filtered_value(value):
+    if type(value) is str and value == Transformer.NOT_PROVIDED:
+        return ""
+    return value
+
+
+@click.command()
+@click.option(
+    "-d",
+    "--db_url",
+    default=None,
+    help="The SQLite database file URL for storage.  Doesn't need to exist beforehand.",
+)
+@click.option("-s", "--solr_url", default=None, help="Solr index URL")
+@click.option(
+    "-v",
+    "--verbosity",
+    default="DEBUG",
+    help="Specify logging level",
+    show_default=True,
+)
+@click.option(
+    "-H", "--heart_rate", is_flag=True, help="Show heartrate diagnostics on 9999"
+)
+@click.option(
+    "-a",
+    "--authority",
+    default=None,
+    help="Which authority to use when selecting and dumping the resolved_content",
+)
+@click_config_file.configuration_option(config_file_name="isb.cfg")
+@click.pass_context
+def main(ctx, db_url, solr_url, verbosity, heart_rate, authority):
+    isb_lib.core.things_main(ctx, db_url, solr_url, verbosity, heart_rate)
+    engine = create_engine(ctx.obj["db_url"], echo=False)
+    SQLModel.metadata.drop_all(engine, tables=[ISBCoreRecord.__table__])
+    SQLModel.metadata.create_all(engine, tables=[ISBCoreRecord.__table__])
+    session = Session(engine)
+    batch_size = 50000
+    session_commit_frequency = 50000
+    rsession = requests.session()
+    iterator = ISBCoreSolrRecordIterator(rsession, authority, batch_size, 0, "id asc")
+    num_records = 0
+    for solr_record in iterator:
+        new_record = ISBCoreRecord()
+        for key, value in SOLR_TO_SQLITE_FIELD_MAPPINGS.items():
+            solr_value = solr_record.get(key)
+            if type(solr_value) is list:
+                filtered_solr_values = [_filtered_value(value) for value in solr_value]
+                solr_value = "; ".join(filtered_solr_values)
```

Comment:
For solr lists, do two things:

(1) Remove any embedded "Not Provided" strings
(2) Turn the list into a semicolon separated string.  SQLite doesn't have a list data type.  Not going to worry about trying to do anything goofy like make a join table for these.

@datadavev if you favor a different approach let me know

Reply:
Replace "Not Provided" with nulls / empty. 

Wrt multivalued fields, sqlite does have JSON support, so could use a JSON field type with a list. Semi-colon delimited is a good option also, but perhaps start with a JSON list and see what sorts of barriers that raises wrt to consumption. If we can do lists, then that opens a lot of flexibility.